data/theme/gnome-shell-sass/_endless.scss

```diff
@@ -959,9 +959,13 @@ $clubhouse-image-height: 187px;
     font: 15px "Metropolis Medium, Sans-Serif";
     box-shadow: 0 2px 4px 4px rgba(0, 0, 0, .4);
     text-shadow: 0 0.5px 2px rgba(0, 0, 0, .4);
-    .next {
-        border-radius: 30px;
-        padding: 15px 20px;
+    &.next {
+        width: 50px;
+        height: 50px;
+        background-image: url("resource:///org/gnome/shell/theme/clubhouse-notification-button-next.svg");
```

Comment:
There is a visual issue in this CSS as can be seen in the screen capture below: If I set a background image, the shadow gets glitchy. If I comment the background image, the shadow looks good. Any idea what's going on? If not I can replace the SVG with a PNG and draw the shadow inside the image.
![captura de pantalla de 2018-12-11 19-58-59](https://user-images.githubusercontent.com/83944/49836523-affcf800-fd81-11e8-886c-f28069cd0539.png)